Wikipedia/Code/AboutViewController.m

```diff
@@ -80,6 +80,18 @@ - (void)wmf_setTextFontColor:(WMFTheme *)theme {
     [self evaluateJavaScript:fontColorJS completionHandler:nil];
 }
 
+- (void)wmf_setLogoStyleWithTheme:(WMFTheme *)theme {
+    // White logo on Dark mode
+    // Black logo on Default and Sepia modes
+    if ([theme.name isEqualToString:WMFTheme.dark.name]) {
```

Comment:
Oh I forgot you'll need to also use the white logo on darkDimmed mode:

> [theme.name isEqualToString:WMFTheme.darkDimmed.name]

Reply:
![screen shot 2017-09-22 at 11 24 36 am](https://user-images.githubusercontent.com/3143487/30758793-a8667290-9f88-11e7-8147-41ab636e4d8d.png)

![screen shot 2017-09-22 at 11 24 43 am](https://user-images.githubusercontent.com/3143487/30758794-a89aee8a-9f88-11e7-9ec0-6845c8d58049.png)

Reply:
Thanks. I've added it.